network/src/main/java/uk/gov/android/network/client/KtorHttpClient.kt

```diff
@@ -79,6 +83,32 @@ class KtorHttpClient(
         }
     }
 
+    override fun setAuthenticationProvider(authenticationProvider: AuthenticationProvider) {
+        this.authenticationProvider = authenticationProvider
+    }
+
+    override suspend fun makeAuthorisedRequest(
+        apiRequest: ApiRequest.Post<*>,
+        scope: String
```

Comment:
We're not using the scope here?

Reply:
the `scope` is being passed on line 94 to the `authenticationProvider`